### DISCUSSION TITLE
What is the product security map upload interface?

### DISCUSSION THREAD

**Discussion by linkin2018** [amzn/selling-partner-api-models#3590] [General]
Dear seller.
Hello!
When selling items online, you must provide and display item security labels, warnings, and logos on the item detail page. If we find that an item lacks the required labels, warnings or logos, we may remove the item. Such information should include six images to show the packaging and all sides of the labels, warnings, and logos. If the seller is a reseller, these images may need to be provided by the United Church Tour supplier or manufacturer.
Here are the two methods that are now applicable for uploading product safety images on the seller's platform
Method -:Use [Upload Image] to add product safety images in bulk for existing products 1. In your control panel, go to [Catalog], then select [Upload Image]; 2. In the upper left corner, select [Bulk Upload Image] to enter the image editing page.
,3. In the [Upload Pictures] box, add pictures or zip files from your computer by dragging or clicking on them. If you add a picture by clicking, please select a picture from your computer and click [Open].
Note:You must name the image like ASIN.VARIANT (e.g.:BO01234567.PSO1) before uploading. After attaching the file, the seller can click the "Submit Image" button for security and compliance "The relevant codes are PSO1,PSO2,PS03,PSO4,PSO5,PSO6. This operation will not overwrite your previously uploaded partial images (PTO1,PTO...). .
Method 2: Use [Upload Image] to add product security images for existing products one by one
1. In your control panel, go to [Catalog], then select [Upload Images]; 2. Select [Image Manager] in the upper left corner to enter the image editing page.
3. Select ASIN from the list in the left column (Note: you must first create a list in the catalog for the ASIN/product to be available in the column).
4. Select "Product Security and Compliance" related codes PSO1,PSO2,PSO3,pSO4,PSO5,pPSO6 from the drop-down menu below the camera icon and click on the camera icon to upload the image; Note: Uploading a product security and compliance image (PSO1,PSO.....) will not leave you with a missing partial image (PSO1,PSO.....). It doesn't make you missing partial pictures (PTO1,PTO.....) In the upload window, you can click the [Refresh] image page in the upper right corner.
Note:If the file name format is not correct, the image image will not be displayed on the product product details detail information, page. Amazon will check the image for compliance and if you t he image is not compliant, the system will notify you to upload a compliant image. For information about these requirements, please refer to the help page "Image Issues".
https://sellercentral-europe.amazon.com/help/hub/reference/external/GT8RSE9S9NK4LP6Z
Once your images have been processed and verified as compliant, they will be posted on the product detail page. This process may take a few days. If you have additional questions about this, you can go to the help page at the link below.
https://sellercentral-europe.amazon.com/help/hub/reference/external/GUH6FA4XSJ2LZFLY?ref=PSl_email_PSIHP

Translated with www.DeepL.com/Translator (free version)

**Comment by sungolivia** [comment]
@linkin2018 ????
